preview.geotiff/ncsa.geo.tiff.py

```diff
@@ -70,9 +70,19 @@ def check_message(self, connector, host, secret_key, resource, parameters):
     # Process the file and upload the results
     def process_message(self, connector, host, secret_key, resource, parameters):
         self.logger = logging.getLogger(__name__)
-
+        isTiffType = False
+        if parameters.get('source'):
+            mimetype = parameters.get('source').get('mimeType')
+            if mimetype == 'image/tiff' or mimetype == 'image/tif':
+                isTiffType = True
         """Process the geotiff and create geoserver layer"""
```

Comment:
this is the python doc string, and there should not be code between this and the def

Reply:
fixed